pkg/service/api_service_test.go

```diff
@@ -74,7 +74,7 @@ var (
 			TenantRef: corev1.LocalObjectReference{
 				Name: tenantA.Name,
 			},
-			Facts: &synv1alpha1.Facts{
+			Facts: synv1alpha1.Facts{
```

Comment:
Did the type change or why are these changes necessary?

Reply:
Yes, the types changed. Sounds like that was not intentional..  

But honestly I think it makes more sense without a pointer anyway.